obsreport/obsreport_test.go

```diff
@@ -44,6 +44,15 @@ const (
 
 var (
 	errFake = errors.New("errFake")
+
+	// Names used by the metrics and labels are hard coded here in order to avoid
+	// inadvertent changes: at this point changing metric names and labeles should
+	// be treated as a breaking changing and requires a good justification.
```

Comment:
Can you please also tell why it is important (i.e. external systems/dashboards that depend on metric names and labels)?